status-im/status-react
# Composer collapsing when editing canceled/done

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

fixes #17433 
fixes #17825

### Summary

Fixed issues:
1. The composer not collapsing when the user cancels editing a message (by pressing x)
2. The keyboard not hiding when the user cancel editing
3. The composer not expanding when the user starts editing a message

#### Platforms


- Android
- iOS

#### Areas that maybe impacted


##### Functional

- 1-1 chats
- public chats
- group chats

### Steps to test


- Open Status
- Open chat
- Select own message and press "Edit message"
- The composer should expand (issue 3.)
- Press x to cancel (issues 1. 2.)

status: ready
